### PR TITLE
Added `tf:"sensitive"` tag

### PR DESCRIPTION
**Pull request by nfx**
Easily mark fields as sensitive

### CODE REVIEW

File: common/reflect_resource.go [hunk 1]
```diff
@@ -112,6 +112,16 @@ func handleForceNew(typeField reflect.StructField, schema *schema.Schema) {
 	}
 }
 
+func handleSensitive(typeField reflect.StructField, schema *schema.Schema) {
+	tfTags := strings.Split(typeField.Tag.Get("tf"), ",")
+	for _, tag := range tfTags {
+		if tag == "sensitive" {
+			schema.ForceNew = true
```
Comment (alexott) [inline common/reflect_resource.go:119]: `schema.Sensitive = true` ? ;-)

**Review by alexott — CHANGES_REQUESTED** [review]

**Comment by nfx** [comment]
Oops

File: common/reflect_resource.go (hunk 1 above)
Comment (nfx) [reply common/reflect_resource.go]: Oops

**Review by alexott — APPROVED** [review]
lgtm

**Pull request merged**